### ISSUE TITLE
Importing angular-highlight-js breaks import of highlight.js

### ISSUE THREAD

**Issue by sqlsolver** [mattlewis92/angular-highlight-js#1]
Webpack build runs fine for 'import * as hljs from 'highlight.js' but after including the import of HighlightJsModule the build fails to find highlight.js. I started with your directive, couldn't get it to work so commented out and worked on just highlight.js (build and typescript definition files), then tried the directive again.
![img_0016](https://cloud.githubusercontent.com/assets/4545063/24390187/02b6db10-133c-11e7-803a-3183765d724b.PNG)
Commenting out directive import allows the build to complete successfully.  I saw you'd updated the NPM package four days ago so I removed, cleaned cache and reinstalled before opening this issue.

**Comment by mattlewis92** [comment]
I couldn't reproduce the error you're experiencing on my mac, however there were a couple things wrong that I fixed in the PR I just sent your repo: https://github.com/sqlsolver/angularSolver/pull/1 Hope that helps!

**Comment by sqlsolver** [comment]
Thanks very much for the PR - I did merge it but unfortunately ' npm run build' immediately fails as shown above when angular-highlight-js is reintroduced. I've checked package.json for the app and both the highlight.js and angular-highlight-js modules and I see no obvious reason why, when angular-highlight-js is added into the mix webpack fails to find highlight.js in node_modules.  Any further thoughts you have are welcome, and I'll update here if I resolve.
